hl7_transformer/tests/test_datetime_transformer.py

```diff
@@ -34,6 +34,34 @@ def test_incorrect_datetime(self):
         with self.assertRaises(ValueError):
             transform_datetime("2023-02-30 12:00:00")  # Invalid date
 
+class TestDateOfDeathTransformer(unittest.TestCase):
+    
+    def test_resurrec_transformation(self) -> None:
+        # Test exact case
+        self.assertEqual(transform_date_of_death("RESURREC"), '""')
+        
+        # Test case insensitive matching
+        self.assertEqual(transform_date_of_death("resurrec"), '""')
+        self.assertEqual(transform_date_of_death("Resurrec"), '""')
+        self.assertEqual(transform_date_of_death("RESURREC"), '""')
+        
+        # Test with whitespace
+        self.assertEqual(transform_date_of_death("  RESURREC  "), '""')
+        self.assertEqual(transform_date_of_death("\tRESURREC\n"), '""')
```

Comment:
you're using subtest in the next test already, this one is also a good candidate due to its repetitiveness:
```suggestion
resurrec_variants = [
        "RESURREC",
        "resurrec",
        "Resurrec",
        "  RESURREC  ",
        "\tRESURREC\n"
    ]
    for variant in resurrec_variants:
        with self.subTest(date=variant):
            self.assertEqual(transform_date_of_death(variant), "")
```